addons/crm_mail_plugin/controllers/mail_plugin.py

```diff
@@ -38,6 +38,7 @@ def _fetch_partner_leads(self, partner, limit=5, offset=0):
                 'expected_revenue': formatLang(request.env, lead.expected_revenue, monetary=True,
                                                currency_obj=lead.company_currency),
                 'probability': lead.probability,
+                'company_id': lead.company_id.id
```

Comment:
Linting: you can add a ',' at the end, so that when adding a new line you don't modify existing lines, just add a new one :)